src/python/ensembl/io/genomio/assembly/download.py

```diff
@@ -193,10 +240,16 @@ def _download_file(
 
 def get_files_selection(dl_dir: Path) -> Dict[str, str]:
     """
-    Among all the files downloaded, only keep a subset for which we use a controlled name.
-    Return a dict[name] = file_path
-    The file_path is relative to the download dir
-    Current names are defined in _FILE_ENDS
+    """Returns a dictionary with the relevant downloaded files classified.
```

Comment:
Forgot to remove the `"""` from the previous line.

Reply:
Removed the extra triple quote thanks @JAlvarezJarreta